assets/js/localization.js

```diff
@@ -44,10 +51,37 @@ $(document).ready(function () {
     });
 
     $('.localeSelect').change(function () {
+
         locale = $(this).val();
         sendEvent('localization', 'localize', locale);
         localizeEverything(false);
         persistChoices('localization');
+
+        var localLang = iso639Codes[locale];
+        var newSrc = recaptchaScriptSrc + 'en';
+        var backoff = false;
+
+        for (var i = 0; i < localizeRecaptcha.length; i++) {
+            if (localLang == localizeRecaptcha[i]) {
+                newSrc = recaptchaScriptSrc + localLang;
+                backoff = false;
+                break
+            }
+            else {
+                backoff = true;
```

Comment:
I think that instead of doing this logic, you can make `backoff = true` on L62 and then remove this `else` branch. Right?